torchao/quantization/quant_primitives.py

```diff
@@ -174,6 +212,31 @@ def quantize_affine(
     Output:
       quantized tensor with requested dtype
     """
+    return _quantize_affine(
+        input,
+        block_size,
+        scale,
+        zero_point,
+        output_dtype,
+        quant_min,
+        quant_max,
+        zero_point_domain.name,
+    )
+
+
+@register_custom_op(quant_lib, 'quantize_affine(Tensor input, int[] block_size, Tensor scale, Tensor? zero_point, ScalarType output_dtype, int? quant_min=None, int? quant_max=None, str zero_point_domain="INT") -> Tensor')
```

Comment:
Are there any utils that take python fn signature and return the string. Otherwise you will have to make sure that string is consistent with fn declaration

Reply:
I'm glad you asked! The answer is yes. https://github.com/pytorch/pytorch/blob/87693b534c09a0a618bb7e757f73dde1b942a854/torch/_library/infer_schema.py#L10

Reply:
Thats awesome. We should definitely use that

Reply:
good point, let me try `infer_schema`